syntaxes/haml.json

```diff
@@ -291,7 +291,7 @@
           "name": "keyword.control.filter.haml"
         }
       },
-      "end": "^(?!\\1  )",
+      "end": "(?m:(?<=\\n)(?!\\1  |\\n))",
```

Comment:
I wasn't actually sure why this regex was different from others. Not sure if I've done the right thing here.